org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--device=all",
+    "--filesystem=xdg-documents",
+    "--filesystem=xdg-download"
+  ],
+  "modules": [
+    "shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json",
+    {
+      "name": "cuda-toolkit",
+      "only-arches": [
+        "x86_64"
+      ],
+      "cleanup": [
+        "/cuda"
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir /app/cuda",
+        "sh cuda_toolkit.run --silent --toolkit --toolkitpath=/app/cuda",
+        "mv /app/cuda/lib64/libcudart.* /app/lib/",
+        "mv /app/cuda/lib64/libcublas* /app/lib/",
```

Comment:
you probably don't need these two if you only need libnvrtc.

Reply:
In addition to the comment above, this wildcard globbing you're using ends up also including archive libraries (`.a` files), which are really big and likely not needed in your case, since you're not statically linking anything...

Reply:
In addition to the comment above, this wildcard globbing you're using ends up also including archive libraries (`.a` files), which are really big and likely not needed in your case, since you're not statically linking anything...

Reply:
great, I don't need it, removed, thanks for the suggestion.